pom.xml

```diff
@@ -281,6 +281,8 @@ limitations under the License.
 
     <!-- Libraries version that MUST be used in all children project -->
 
+    <current-year>2017</current-year>
+    <rudder-major-version>4.1</rudder-major-version>
     <rudder-version>4.1.0~beta4-SNAPSHOT</rudder-version>
```

Comment:
Yes, and it probably won't work because this is the probable reason of the existence of the next line.